### ISSUE TITLE
Can't create patches if there are also shell scripts used in preinstall/postinstall

### ISSUE THREAD

**Issue by dawnmist** [ds300/patch-package#52]
If there are additional actions required during normal pre/post install for the program, the patch-package tries to run those as well as part of the build new package process. However, because it has not copied them to the temp build directory, those scripts cannot be found so the build fails.

For example:
./util/pre-install.sh
```sh
#!/bin/bash
if [ ! -f "${PWD}/.env"]
then
    echo "REACT_APP_DEMO_MODE=true"
fi
```
package.json:
```json
{
  "scripts": {
    "preinstall": "./util/pre-install.sh",
    "postinstall": "patch-package"
  }
}
```

The failure log states that the "./util/pre-install.sh" script was not found in the temp directory.

**Comment by ds300** [comment]
Hi! :wave: Thanks for the report. That makes perfect sense. The scripts entry should probably be deleted from the package.json copy that is used in the temp directory.

I'll try to take a look at doing that by tomorrow evening.

**Comment by ds300** [comment]
Hi again! I forgot that I already tackled this issue in 6.0.0-3 (#47) but apparently I made a stupid mistake and it didn't actually fix the problem. I've tried again, and this time I added a regression test for extra certainty :D

It's available in 6.0.0-5 — Thanks for your patience! :bow:
